Alignment/HIPAlignmentAlgorithm/scripts/configure.pl

```diff
@@ -2,17 +2,13 @@
 use File::Basename;
```

Comment:
@usarica this is an old script, but my comment on `batch.py` holds true also in this case.